src/poliastro/atmosphere/coesa62.py

```diff
@@ -189,10 +208,9 @@ def density(self, alt, geometric=True):
         # Check if valid range and convert to geopotential
         z, h = self._check_altitude(alt, r0, geometric=geometric)
 
-        # TODO: implement atmosphere up to 1000km
-        if z > 90 * u.km:
+        if z > 700 * u.km:
             raise NotImplementedError(
-                "Density in COESA62 has just been implemented up to 90km."
+                "Density in COESA62 has just been implemented up to 700km."
```

Comment:
This atmospheric model has a maximum altitude of 700km. We should raise a `ValueError` instead of a `NotImplementedOne`.